api/src/opentrons/protocol_api/instrument_context.py

```diff
@@ -1856,31 +1856,58 @@ def resin_tip_unseal(
 
         return self
 
-    @requires_version(2, 22)
+    @requires_version(2, 23)
     def resin_tip_dispense(
         self,
         location: types.Location,
         volume: Optional[float] = None,
         rate: Optional[float] = None,
     ) -> InstrumentContext:
-        """Dispense a volume from resin tips into a labware.
+        """Push liquid out of resin tips that are currently sealed to a pipette.
+
+        The ``location`` provided is where the liquid will be dispensed into.
+
+        The volume and rate parameters for this function control the motion of the plunger
+        to create a desired pressure profile inside the pipette chamber. Unlike a regular
+        dispense action, the volume and rate do not correspond to liquid volume or flow rate
+        dispensed from the resin tips. Select your values for volume and flow rate based on
+        experimentation with the resin tips to create a pressure profile.
 
-        The location provided should contain resin tips labware as well as a
-        receptical for dispensed liquid. Dispensing from tip will perform a
-        `dispense` action of the specified volume at a desired flow rate.
+        The common way to use this function is as follows:
+        #. Seal resin tips to the pipette using :py:meth:`InstrumentContext.resin_tip_seal`.
 
-        :param location: A location containing resin tips.
+        #. Use :py:meth:`InstrumentContext.resin_tip_dispense` to displace an experimentally
+           derived volume at an experimentally derived rate to create an experimentally derived
+           target pressure inside the pipette.
+
+        #. Use :py:meth:`ProtocolContext.delay` to wait an experimentally derived amount of
+           time for the pressure inside the pipette to push liquid into and through the resin tip
+           and out the other side.
+
+        #. As liquid passes through the resin tip, the pressure inside the pipette will
+           fall. If not all liquid has been dispensed from the resin tip, repeat steps 2
+           and 3.
+
+        #. Unseal resin tips from the pipette using :py:meth:`InstrumentContext.resin_tip_unseal`.
+
+        Flex pipette pressure sensors will raise an overpressure when a differential pressure
+        inside the pipette chamber above sensor limits is detected. You may need to disable the
+        pressure sensor to create the required pressure profile, which can cause damage or
+        failure to your pipette's pressure sensors.
+
+        :param location: The location into which to dispense.
         :type location: :py:class:`~.types.Location`
 
-        :param volume: Will default to maximum, recommended to use the default.
-                       The volume, in µL, that the pipette will prepare to handle.
+        :param volume: The volume that the plunger should displace. Does not directly relate
+                       to the volume of liquid that will be dispensed.
```

Comment:
It's still µL of air to move, and presumably the max is governed by the nominal capacity of the tip in µL.
```suggestion
        :param volume: The volume that the plunger should displace, in µL. Does not directly relate
                       to the volume of liquid that will be dispensed.
```

Reply:
the max is governed by the physical displacement of the plunger, actually - the number in the pipette name.